site/source/tutorials/008_animations/demo/finished/zombies/src/main.ts

```diff
@@ -0,0 +1,12 @@
+import { ProjectorMixin } from '@dojo/widget-core/mixins/Projector';
+import Zombies from './widgets/Zombies';
+
+// Create a projector to convert the virtual DOM produced by the application into the rendered page.
+// For more information on starting up a Dojo 2 application, take a look at
```

Comment:
Maybe setting up instead of starting up? Starting up to me sort of implies starting an automation task.

Reply:
worth raising this in `cli-create-app` too then